### PR TITLE
add additonal params to the registration message

### PR DESCRIPTION
**Pull request by rivernate**

### CODE REVIEW

File: lib/ims/lti/models/messages/registration_request.rb [hunk 1]
```diff
@@ -12,7 +13,7 @@ def initialize(attrs = {})
     end
 
     def generate_key_and_password
-      self.reg_key, self.reg_password = 2.times.map { SecureRandom.uuid }
```
Comment (augiethornton) [inline lib/ims/lti/models/messages/registration_request.rb]: I'm a fan of the white space here for clarity :( it's the JS way!

File: spec/ims/lti/models/messages/registration_request_spec.rb [hunk 2]
```diff
@@ -45,7 +46,7 @@ module IMS::LTI::Models::Messages
       end
 
       it 'returns the key and secret' do
-        expect(subject.generate_key_and_password).to eq 2.times.map{ 'random_uuid' }
+        expect(subject.generate_key_and_password).to eq 2.times.map {'random_uuid'}
```
Comment (augiethornton) [inline spec/ims/lti/models/messages/registration_request_spec.rb]: I don't mind the space after `map` however, it's nice to have the whitespace between the string and brackets. I lean on the Javascript preference of it.

File: lib/ims/lti/models/messages/registration_request.rb (hunk 1 above)
Comment (rivernate) [inline lib/ims/lti/models/messages/registration_request.rb]: done

File: spec/ims/lti/models/messages/registration_request_spec.rb (hunk 2 above)
Comment (rivernate) [inline spec/ims/lti/models/messages/registration_request_spec.rb]: done

**Pull request merged**